stack.yml

```diff
@@ -139,6 +139,8 @@ functions:
       write_debug: true
       read_debug: true
       combine_output: false
+      secure_connection: false
```

Comment:
These should be in the same file as the ones containing `s3_url`. You also need to update the README.md to give AWS-specific instructions.